worksheet_D/oxo.py

```diff
@@ -0,0 +1,107 @@
+class OxoBoard:
+    def __init__(self):
+        # Initialising board array
+        self.boardList = [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
```

Comment:
OK, but could be made neater by initialising with e.g. list comprehensions or multiplication